arches/app/models/fields/i18n.py

```diff
@@ -242,19 +243,6 @@ def __init__(self, value=None, lang=None, use_nulls=False, attname=None):
     def _parse(self, value, lang, use_nulls):
         ret = {}
 
-        if isinstance(value, Cast):
-            # Django 4.2 regression: bulk_update() sends Cast expressions
-            # https://code.djangoproject.com/ticket/35167
```

Comment:
This was fixed in Django 5.2, see https://github.com/django/django/pull/19183, so this check is no longer necessary.